web/src/components/__tests__/netflow-traffic.spec.tsx

```diff
@@ -3,6 +3,17 @@ import { shallow } from 'enzyme';
 
 import NetflowTraffic from "../netflow-traffic";
 
+jest.mock("react-i18next", () => {
+  return {
+    useTranslation: () => {
+      return {
+        t: (s) => s,
+      };
+    },
+  };
+});
+
+
```

Comment:
This should be in a separate file called by [setupfilesafterenv](https://jestjs.io/docs/configuration#setupfilesafterenv-array) jest config parameter in `package.json` to avoid copy / past in all `spec.tsx` files